Assets/Plugins/LeapMotion/Modules/Hands/Scripts/HandBinder/HandBinderData.cs

```diff
@@ -36,6 +36,12 @@ public class TransformStore {
         public Vector3 rotation = Vector3.zero;
     }
 
+    [System.Serializable]
+    public class SerializedTransform {
+        public TransformStore transform;
+        public GameObject reference;
+    }
+
```

Comment:
Could have a look at storing a Matrix4x4 instead of this approach to serialize transform data.